test/aggregators/aggregators.js

```diff
@@ -47,6 +47,14 @@ describe('Aggregators', () => {
     });
   });
 
+  it('has no aggregators failing when shortName is called', () => {
+    return walkTree.then(aggregators => {
+      aggregators.forEach(aggregator => {
+        assert.doesNotThrow(_ => aggregator.shortName);
```

Comment:
Well the base class for Aggregator throws. All we want to check is that they've overridden it in some way. I guess we could `.ok` as an inverse.